IntegrationTests/src/bkr/inttest/server/test_model.py

```diff
@@ -1400,6 +1400,26 @@ def test_ksmeta_ks_keyword(self):
         self.assertIn('preseed/url=http://', r1.installation.kernel_options)
         self.assertNotIn('ks=', r1.installation.kernel_options)
 
+    def test_ks_option_for_older_systems(self):
+        distro_tree = data_setup.create_distro_tree(osmajor=u'CentOS7')
+        recipe = self.provision_distro_tree(distro_tree)
+        self.assertIn('ks=', recipe.installation.kernel_options)
+        self.assertNotIn('inst.ks=', recipe.installation.kernel_options)
+
+    def test_inst_ks_option_for_newer_systems(self):
+        distro_tree = data_setup.create_distro_tree(osmajor=u'Fedora34')
+        recipe = self.provision_distro_tree(distro_tree)
+        self.assertIn('inst.ks=', recipe.installation.kernel_options)
```

Comment:
Similar to older_systems test, should we test that 'ks=' is assertNotIn()???

Reply:
Well if 'inst.ks=' is there, 'ks=' will be there as well...

Reply:
asserted with a regex